src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -70,8 +70,8 @@ void setConversationStore(ConversationStore conversationStore) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    List<Conversation> conversations = conversationStore.getAllConversations();
-    request.setAttribute("conversations", conversations);
+	List<Conversation> publicConversations = conversationStore.getAllPublicConversations();
+	request.setAttribute("publicConversations", publicConversations);
     request.getRequestDispatcher("/WEB-INF/view/conversations.jsp").forward(request, response);
```

Comment:
For the group conversations, we could create a list that is groupConversations using this same idea.

Reply:
Yeah I was thinking the same thing! That's what I was going to do in my next PR.